index.js

```diff
@@ -1,4 +1,19 @@
 'use strict'
 
 const binding = require('node-gyp-build')(__dirname)
-module.exports = binding.CuckooFilter
+
+function CuckooFilter (totalSize, bitsPerItem = 12) {
```

Comment:
I would throw if this was called with new.

Reply:
Err okay, was trying to preserve backwards compatibility with the `new Cuckoofilter(totalItems)` API.

Reply:
Another thing you can do is:

```js
module.exports = CuckooFilter32
CuckooFilter32.CuckooFilter32 = CuckooFilter32
CuckooFilter32.CuckooFilter16 = CuckooFilter16
// ...
```

So the interface stays the same.